src/ut/call_list_store_processor_test.cpp

```diff
@@ -118,12 +123,14 @@ class CallListStoreProcessorWithLimitTest : public ::testing::Test
     delete _cls; _cls = NULL;
     delete _stats_aggregator; _stats_aggregator = NULL;
     delete _clsp; _clsp = NULL;
+    delete _http_notifier; _http_notifier = NULL;
   }
 
   StrictMock<MockLoadMonitor> _load_monitor;
   CallListStoreProcessor* _clsp;
   MockCallListStore* _cls;
   LastValueCache* _stats_aggregator;
+  MockHttpNotifier* _http_notifier;
 };
 
 // Create a vector of call list store fragments that the mock
```

Comment:
Also do you have any tests that the notifier is not called if an empty/bad string is passed as the notify URL?